tests/lib/e2e/users.fixtures.ts

```diff
@@ -0,0 +1,43 @@
+import { User } from '../../../lib/core/users/User';
+import { v4 } from 'uuid';
+
+export type MongoUserModel = Required<Omit<User, 'id'>> & {
+  _id: string;
+};
+
+const formatUser = ({ _id, ...model }: MongoUserModel): User => ({
+  ...model,
+  id: _id,
+});
+
+const usersTest: MongoUserModel[] = [
+  {
+    _id: v4(),
+    hashpass:
+      '4b796e7bbe57f7092d3627d3fa7e6f645b0e50e3411e1fafd61dbf27241d836d',
+    subscriptionPlan: {
+      isSubscribed: false,
+    },
+    referralPartner: null,
+    email: 'fff@ff.com',
```

Comment:
use [chance](https://github.com/internxt/drive-server-wip/blob/59e13c6ef5a12e5aa89317e9f8eb72fd1165cfbc/test/fixtures.ts#L2) as we do use on drive-server-wip to generate random values with the most realistic format possible.